Marlin/src/pins/rambo/pins_EINSY_RAMBO.h

```diff
@@ -179,30 +208,30 @@
   #if IS_ULTIPANEL || TOUCH_UI_ULTIPANEL
 
     #if ENABLED(CR10_STOCKDISPLAY)
-      #define LCD_PINS_RS                     85
-      #define LCD_PINS_ENABLE                 71
-      #define LCD_PINS_D4                     70
-      #define BTN_EN1                         61
-      #define BTN_EN2                         59
+      #define LCD_PINS_RS            EXP1_07_PIN
+      #define LCD_PINS_ENABLE        EXP1_08_PIN
+      #define LCD_PINS_D4            EXP1_06_PIN
+      #define BTN_EN1                EXP1_03_PIN
+      #define BTN_EN2                EXP1_05_PIN
     #else
-      #define LCD_PINS_RS                     82
-      #define LCD_PINS_ENABLE                 61
-      #define LCD_PINS_D4                     59
-      #define LCD_PINS_D5                     70
-      #define LCD_PINS_D6                     85
-      #define LCD_PINS_D7                     71
-      #define BTN_EN1                         14
-      #define BTN_EN2                         72
+      #define LCD_PINS_RS            EXP1_04_PIN
+      #define LCD_PINS_ENABLE        EXP1_03_PIN
+      #define LCD_PINS_D4            EXP1_05_PIN
+      #define LCD_PINS_D5            EXP1_06_PIN
+      #define LCD_PINS_D6            EXP1_07_PIN
+      #define LCD_PINS_D7            EXP1_08_PIN
+      #define BTN_EN1                EXP2_05_PIN
+      #define BTN_EN2                EXP2_03_PIN
 
       #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
         #define BTN_ENC_EN           LCD_PINS_D7  // Detect the presence of the encoder
```

Comment:
This one should be changed to `EXP1_08_PIN`

Reply:
To late already merged...  looking at all other BTN_ENC_EN, they are all set to LCD_PINS_D7 or a pin, none are set to a EXP name...